lstchain/io/__init__.py

```diff
@@ -29,6 +29,7 @@
     get_srcdep_assumed_positions,
     get_srcdep_params,
     add_source_filenames,
+    remove_duplicate_events,
```

Comment:
better to name it `remove_duplicated_events`

Reply:
Why not duplicate?

Reply:
`duplicate` can be noun, verb or adjective, I think that with `duplicated` it is clearer, but fine